aiomisc/plugins/__init__.py

```diff
@@ -1,24 +1,30 @@
 import logging
 import os
+from itertools import chain
 from types import MappingProxyType
 from typing import Callable, Mapping
 
+from aiomisc.compat import entry_pont_iterator
+
 
 def setup_plugins() -> Mapping[str, Callable]:
     if os.getenv("AIOMISC_NO_PLUGINS"):
         return MappingProxyType({})
 
-    import pkg_resources
-
     plugins = {}
+    logger = logging.getLogger(__name__)
 
-    for entry_point in pkg_resources.iter_entry_points("aiomisc.plugins"):
-        plugins[entry_point.name] = entry_point.load()
-
-    for entry_point in pkg_resources.iter_entry_points("aiomisc"):
-        plugins[entry_point.name] = entry_point.load()
+    for entry_point in chain(
+        entry_pont_iterator("aiomisc.plugins"),
+        entry_pont_iterator("aiomisc"),
```

Comment:
I don't like this hardcode.
Maybe it's better to use `aiomisc.__name__` and `aiomisc.plugins.__name__` instead?

Reply:
`explicit is better than implicit`